CHANGELOG.md

```diff
@@ -0,0 +1,260 @@
+# Change Log
+
+All notable changes to this project will be documented in this file. This
+project adheres to [Semantic Versioning](http://semver.org).
+
+## [Unreleased]
+
+### Added
+
+- Added core grid functionality in `grid-column`
+- Added improved documentation and releasing information
+- Added `grid-container` for floated grid which contains a simple clearfix
+- Added `bundle exec` for rake release task
+- Added `grid-push` functionality
+- Added `neat-setting-retrieve()` function
+- Added `grid-calc` to abstract the calculation of column widths
+
+### Changed
+
+- Change to https for RubyGems gem source
+- Update gulpfile to watch contrib styles and not just core project files
+- Use `core/` as the primary Neat source directory
+
+### Removed
+
+- Removed all previous Neat Sass
+- Removed all tests
+- Removed Sache
+- Removed Travis
+- Removed engine, Railtie, and install task
```

Comment:
We can remove this, as it is not notable/user-facing. What this does mean, though, is that our Rails requirement has been increased. In Bourbon, we added “Dropped support for Ruby on Rails versions older than 4.2.”.